{
  "containers": {
    "adp": [
      {
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-04T04:32:13.466Z"
        },
        "title": "CVE Program Container",
        "references": [
          {
            "tags": [
              "x_refsource_MISC",
              "x_transferred"
            ],
            "url": "http://lua-users.org/lists/lua-l/2021-11/msg00186.html"
          },
          {
            "tags": [
              "x_refsource_MISC",
              "x_transferred"
            ],
            "url": "http://lua-users.org/lists/lua-l/2021-12/msg00007.html"
          },
          {
            "tags": [
              "x_refsource_MISC",
              "x_transferred"
            ],
            "url": "http://lua-users.org/lists/lua-l/2021-12/msg00015.html"
          },
          {
            "tags": [
              "x_refsource_MISC",
              "x_transferred"
            ],
            "url": "http://lua-users.org/lists/lua-l/2021-12/msg00030.html"
          },
          {
            "tags": [
              "x_refsource_MISC",
              "x_transferred"
            ],
            "url": "https://github.com/Lua-Project/lua-5.4.4-sandbox-escape-with-new-vulnerability"
          }
        ]
      }
    ]
  },
  "cveMetadata": {
    "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
    "assignerShortName": "mitre",
    "cveId": "CVE-2021-44964",
    "datePublished": "2022-03-14T14:24:51",
    "dateReserved": "2021-12-13T00:00:00",
    "dateUpdated": "2024-08-04T04:32:13.466Z",
    "state": "PUBLISHED"
  },
  "search": "解析验收测试文案"
}